identity/backend/src/main/java/io/camunda/identity/usermanagement/initializer/DefaultUserInitializer.java

```diff
@@ -37,6 +37,7 @@ public void setupUsers() {
   }
 
   private void setupUser(final CamundaUserWithPassword camundaUserWithPassword) {
+    camundaUserWithPassword.setEnabled(true);
```

Comment:
I ran into the issue of Spring complaining about a disabled user when I tried to perform an action using built-in functionality (I think around authorities). Hence, I added this line here.

In general, I think the initial users should be enabled. That makes sense in my reasoning. Feel free to challenge that though.

Reply:
I think this problem is fixed in the main branch, as now enabled is by default set to true in CamundaUserWithPassword.

Reply:
I think I'm too dumb to find where it's fixed 🙈 ... would you mind sharing a link to the place in the code that you're referring to please? 😅

Reply:
Here is the link: [CamundaUser.java](https://github.com/camunda/camunda/blob/71762e82cd48577d780adec88aef1202d01a35f1/identity/common/src/main/java/io/camunda/identity/usermanagement/CamundaUser.java#L16
It was not initiated before but now it's defaulted to true.

Reply:
Ohhh, I see, thank you! :) 
I was in the wrong branch 🙈